### PR TITLE
fixed tooltip popup for title of marker

### PR DESCRIPTION
**Pull request by sackh**
Signed-off-by: Kharude, Sachin <sachin.kharude@here.com>
Fix for #676

### CODE REVIEW

**Comment by sackh** [comment]
@davidbrochart - please review this.

File: js/src/layers/Marker.js [hunk 1]
```diff
@@ -50,6 +50,15 @@ export class LeafletMarkerView extends layer.LeafletUILayerView {
       this.model.set('location', [position.lat, position.lng]);
       this.touch();
     });
+    if (this.model.get('title')) {
+      this.obj.bindPopup(this.model.get('title'));
```
Comment (davidbrochart) [inline js/src/layers/Marker.js:54]: Maybe:
```javascript
title = this.model.get('title');
if (title) {
  this.obj.bindPopup(title);
```

Comment (sackh) [reply js/src/layers/Marker.js:54]: sure I will make this change.

**Comment by davidbrochart** [comment]
Thanks a lot!

**Comment by martinRenou** [comment]
This should be reverted.

**Pull request merged**